### PR TITLE
qa/workunits/rdb/krbd_fallocate.sh: force python3 and fix parameter to rbd create

### PR DESCRIPTION
**Pull request by julpark-rh**
RHEL 8 Pass log:
http://pulpito.ceph.redhat.com/julpark-2020-03-29_22:28:36-krbd:rbd-nomount-nautilus-distro-basic-argo/

Signed-off-by: julpark-rh <julpark@redhat.com>

### CODE REVIEW

**Comment by rakeshgm** [comment]
Please remove `WIP` from the title

**Comment by julpark-rh** [comment]
removed it WIP

File: qa/workunits/rbd/krbd_fallocate.sh [hunk 1]
```diff
@@ -115,7 +115,7 @@ assert_zeroes 0
 # unaligned blkdev_issue_discard
 allocate
 py_blkdiscard $((OBJECT_SIZE / 2))
-assert_zeroes_unaligned $NUM_OBJECTS
```
Comment (rakeshgm) [inline qa/workunits/rbd/krbd_fallocate.sh]: Any reason why you have removed `$NUM_OBJECTS`  here

**Review by rakeshgm — APPROVED** [review]

**Review by rakeshgm — APPROVED** [review]

**Comment by rakeshgm** [comment]
Please Sign your commits.

**Comment by julpark-rh** [comment]
Signed-off-by: julpark-rh julpark@redhat.com

**Pull request merged**